python/lsst/ctrl/ingestd/config.py

```diff
@@ -38,24 +38,28 @@ def __init__(self, filename: str):
         with open(filename) as file:
```

Comment:
Consider migrating to a pydantic model in the future. This will make a lot of this code much easier (and give you validation).